src/RootMaskedTextField/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
+import TextField from 'material-ui/TextField';
+import Divider from 'material-ui/Divider';
+import { MaskedInput } from 'react-text-mask';
+import { nonEditingStyle } from '../styles/styles.js';
+import removeMask from '../validations/removeMask.js';
+
+export default class RootMaskedTextField extends React.Component  {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: props.value,
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    let newValue = event.target.value;
+    console.log(this.state.value);
```

Comment:
@emilrb this code seems good to go, but this log commands must be removed prior to merge.